External/SPMmodified/xASL/xASL_adm_UnixPath.m

```diff
@@ -24,11 +28,20 @@
 % __________________________________
 % Copyright 2021 ExploreASL
         
+    if nargin<2 || isempty(bWSL)
+        bWSL = false;
+    elseif ispc
```

Comment:
This statement is only reached if bWSL is defined. Either TRUE or FALSE - this doesn't really make sense.
Either this needs to be changed, or the input parameter has to change from bWSL to bForceUnix, so that the setting of bWSL below makes sense.

Reply:
I will rename the booleans such that this makes sense. But this does already make sense. You'll see ;)

Reply:
Looks good now.